WindowsFormsApp2/AddExpenseForm.cs

```diff
@@ -33,6 +34,7 @@ public void AddCostButton_Click(object sender, EventArgs e)
             if (Convert.ToString(newCostCategory.SelectedItem) == "Food")
                 cost.Category = "Food";
             cost.Date = newCostDate.Value.Date;
+            cost.Account = expenseAccount;
```

Comment:
Why this variable is called "cost"? Rename it to "expense".

Reply:
I renamed it.